src/wccp2.cc

```diff
@@ -1743,7 +1743,8 @@ wccp2AssignBuckets(void *)
 
         assignment_key = (struct assignment_key_t *) &wccp_packet[offset];
 
-        assignment_key->master_number = htonl(++service_list_ptr->change_num);
+        service_list_ptr->change_num++;
```

Comment:
Feel free to change in whatever way is appropriate. I just wanted to report the problem and suggest a fix; I have no opinion about how it actually gets fixed in the end! =)

Reply:
Question is though does that change still work as well in the build evironment where you found the problem. So if you make the change you can as well confirm its good. Whereas if we do it from here it may need re-fixing back to your version.

Reply:
Yes, prefix increment works.